data.md

```diff
@@ -0,0 +1,106 @@
+## How We Use Data
+
+_The outline and format of this page is adopted from [Projects by IF: How IF uses data](https://www.projectsbyif.com/how-if-uses-data/)._
+
+- [How We Use Data](#how-we-use-data)
+- [Our Websites and Hosted Services](#our-websites-and-hosted-services)
+    - [What data we collect](#what-data-we-collect)
+    - [Why we collect data](#why-we-collect-data)
+    - [Where we keep data](#where-we-keep-data)
+    - [How long we keep data](#how-long-we-keep-data)
+- [Our Social Media Accounts and Third-party Services](#our-social-media-accounts-and-third-party-services)
+
+---
```

Comment:
```suggestion
## About Hypha
```

Reply:
Not the greatest form to use a section header from another page, I think we don't need this as a top level description if it is a graph pointing to our services and domains (maybe that header would be better if there needs to be one)

Reply:
Yea I was struggling to write a header for this section, but there needs to be something otherwise it looks awkward continuing from the Contents.

>maybe that header would be better if there needs to be one

What do you mean? We put this underneath `## Our Websites and Hosted Services`? That sounds ok to me.